scripts/qtcreator/templates/wizards/openframeworks/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ofMain.h"
```

Comment:
i wasn't but it should be pretty straight forward so i'll take a look when i have a moment